src/backend/CassandraBackend.cpp

```diff
@@ -1153,25 +1146,24 @@ CassandraBackend::open(bool readOnly)
         cass_ssl_free(context);
     }
 
-    std::string keyspace = getString("keyspace");
+    auto keyspace = config_.value_or<std::string>("keyspace", "");
     if (keyspace.empty())
     {
         BOOST_LOG_TRIVIAL(warning)
             << "No keyspace specified. Using keyspace clio";
         keyspace = "clio";
     }
 
-    int rf = getInt("replication_factor") ? *getInt("replication_factor") : 3;
-
-    std::string tablePrefix = getString("table_prefix");
+    auto rf = config_.value_or<int>("replication_factor", 3);
+    auto tablePrefix = config_.value_or<std::string>("table_prefix", "");
```

Comment:
I think we should remove this config option. Though maybe not necessarily as part of this PR.